.github/workflows/teamsjs-released.yml

```diff
@@ -4,10 +4,58 @@ on:
     types: [teamsjs-released]
 jobs:
   notification:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest      
```

Comment:
minor: you could remove these redundant spaces

Reply:
Spaces are removed. Thx!